### PR TITLE
#163127144 Build A Rest full Endpoint To Get A Specific Question

### PR DESCRIPTION
**Pull request by Ogollah**
### What does this PR do?
Adds a get a specific questions endpoint  for  QUESTIONER API

### Description of Task to be completed?
Have the following endpoints working
```
|Endpoint                            | Functionality                    |HTTP method 
|------------------------------------|----------------------------------|-------------
|/api/v1/user/signup                 |Signup account                    |POST        
|/api/v1/user/auth/signin            |Signin in user                    |POST
|/api/v1/user/auth/signout           |Signout in user                   |POST
|/api/v1/meetups/create              |Create a meetup record            |POST
|/api/v1/meetups/upcoming            |Get upcoming meetups              |GET
|/api/v1/meetups/<meetup-id>         |Fetch specific upcoming meetup    |GET
|/api/v1/questions/questions         |Fetch all question                |GET
|/api/v1/questions/<meetup-id>/create|Create a question                 |POST
|/api/v1/questions/<question-id> |access specific question                 |GET
```

### How should this be manually tested?
Using your terminal run the following commands:
```
- cd path/to/server-root
- git clone this repo
- cd to Questioner
- python3 -m venv env
- source env/bin/activate
- pip install requirement.txt
- on command line run manage.py test or run
```
### Any background context you want to provide?
No database
### What are the relevant pivotal tracker stories?
[#163127144](https://www.pivotaltracker.com/story/show/163127144)

### CODE REVIEW

File: api/v1/main/service/meetup_service.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def save_new_meetup(meetup_data):
     images = meetup_data["images"]
     Tags = meetup_data["Tags"]
     createdOn = datetime.datetime.utcnow()
-    happeningOn = create_future_date(date_data=meetup_data["happeningOn"])
+    happeningOn = date_data=meetup_data["happeningOn"]
```
Comment (oagutu) [inline api/v1/main/service/meetup_service.py]: I don't think this is necessary, please fix it

File: api/v1/main/service/question_service/question_service.py [hunk 2]
```diff
@@ -68,3 +68,17 @@ def save_new_question(question_data, meetup_id):
...
+    """
+    Get a specific question.
+    """
+    question = get_question_by_id(question_id)
+    if question:
+        return question, 200
+    else:
+        response_object = {
+            'status':404,
+            'message':'Question you are looking for is nologer there.'
```
Comment (oagutu) [inline api/v1/main/service/question_service/question_service.py]: Minor typo: I think you meant `no longer`

**Review by oagutu — COMMENTED** [review]
Nice work on this. Here are a few issues I noticed:
- I just noticed you are not returning the data for successful `POST` requests, eg, when creating a question, is there a reason for this?
- When one gets a question using a` non-existent question_id` , you get a response body with all the data fields as `null`. This should not be the case, instead, the response should simply contain the error message and response status code

**Comment by Ogollah** [comment]
Noted.

**Pull request merged**